client/controller/ssr-setup-locale.js

```diff
@@ -44,14 +45,14 @@ export function ssrSetupLocaleMiddleware() {
 			setLocaleData( cachedTranslations );
 		} else {
 			readFile( getAssetFilePath( `languages/${ lang }-v1.1.json` ), 'utf-8' )
-				.then( ( data ) => {
-					const translations = JSON.parse( data );
+				.then( ( data ) => JSON.parse( data ) )
+				// Fall back to the remote one if the local translation file is not found.
+				.catch( () => getLanguageFile( lang ) )
```

Comment:
We don't build the languages on the dev env so use the remote translation if the locale file is not found